package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-chime-sdk-js",
-  "version": "1.2.5",
+  "version": "1.2.6",
```

Comment:
Do not need to update the version as this PR does not modify Chime SDK itself.

Reply:
This is done by npm run build:release command!

Reply:
Right.. Thanks for checking.